test/unit/search/fm_index_cursor/fm_index_cursor_collection_test_template.hpp

```diff
@@ -306,8 +306,7 @@ TYPED_TEST_P(fm_index_cursor_collection_test, incomplete_alphabet)
 
         EXPECT_TRUE(it.extend_right(this->text4[0]));                                           // 'A'
         EXPECT_TRUE(it.cycle_back());
-        EXPECT_TRUE(std::ranges::equal(it.path_label(this->text_col7),
-                                       seqan3::views::slice(this->text4, 1, 2)));               // "T"
+        EXPECT_RANGE_EQ(it.path_label(this->text_col7), seqan3::views::slice(this->text4, 1, 2)); // "T"
```

Comment:
Small :nail_care:, 299-307 need two extra white spaces for the comments to be aligned.